.ci/jenkins/dsl/jobs.groovy

```diff
@@ -172,6 +172,9 @@ void setupBuildImageJob(JobType jobType, boolean prodCI = false) {
         AUTHOR_CREDS_ID: "${GIT_AUTHOR_CREDENTIALS_ID}",
         AUTHOR_TOKEN_CREDS_ID: "${GIT_AUTHOR_TOKEN_CREDENTIALS_ID}",
     ])
+    if (Utils.hasBindingValue(this, 'DISABLE_DEPLOY')) {
+        jobParams.env.put('DEPLOY_IMAGE', !Utils.getBindingValue(this, 'DISABLE_DEPLOY').toBoolean())
```

Comment:
`DEPLOY_IMAGE` is already a parameter to the job. Should not be defined as env var...
I just noticed that you can simply put https://github.com/kiegroup/kogito-images/blob/main/.ci/jenkins/Jenkinsfile.deploy#L285 to `false` (or dependent on `DISABLE_DEPLOY` value that needs to be passed as env in `setupDeployJob` method) ... That should be enough